service/build.gradle

```diff
@@ -133,6 +133,7 @@ tasks.withType(Test) {
 }
 
 check.dependsOn integrationTest
+integrationTest.mustRunAfter test
```

Comment:
What's this change all about?

I'm not sure I'd consider this a refactoring.

Reply:
I will remove this line